src/components/dialog/dialog.js

```diff
@@ -114,6 +117,20 @@ const VUIDialog = ({
         dialog.setAttribute('tabindex', 1);
         dialog.setAttribute('aria-hidden', false);
 
+        // Alerts and modal dialog types don't have close buttons
+        if (hasCloseBtn && !isAlert && !isModal) {
+            const closeBtnEl = createEl({
+                element: 'button',
+                className: ['js-dialog-btn-close', closeButtonClassName],
+                type: 'button',
+                children: [{
+                    element: 'span',
```

Comment:
It doesn't really *need* it, I was more thinking if the consumer wanted to visually hide the text and replace it with an `X` icon or something this would enable them to do so